IBPSA/Airflow/Multizone/package.order

```diff
@@ -1,10 +1,16 @@
 UsersGuide
+MediumColumn
```

Comment:
The model order was revised, meaning they are no longer alphabetical. This will cause unit tests to fail. The package order should be alphabetical (except for some package names like 'Examples', 'Validation', etc.)